app/services/add_receipt_to_c2_proposal_attributes.rb

```diff
@@ -0,0 +1,21 @@
+class AddReceiptToC2ProposalAttributes
```

Comment:
I can see that. On one hand, it has a `#perform` method and does a tiny bit of data transformation. On the other hand, it is sort of a business logic object. Your call I guess?